### PR TITLE
Madalinap/tema8

### PR DESCRIPTION
**Pull request by pmadalina88**

### CODE REVIEW

File: src/test/java/madalinapopescu/tests/tema8/teste/SimpleDynamicButtonsTests.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+public class SimpleDynamicButtonsTests extends BaseTest{
+    @Test(groups = {"mobile", "desktop"})
+    public void verifyClickOnAllButtons(){
+        driver.get("https://testpages.herokuapp.com/styled/dynamic-buttons-simple.html");
+        SimpleDynamicButtonPage simpleDynamicButtonPages = new SimpleDynamicButtonPage(driver);
+        simpleDynamicButtonPages.clickOnStartButton();
+        simpleDynamicButtonPages.clickOnOneButton();
+        simpleDynamicButtonPages.clickOnSecondButton();
+        simpleDynamicButtonPages.clickOnThertButton();
+//        Assert message All Buttons Clicked apper at the end
```
Comment (95teo) [inline src/test/java/madalinapopescu/tests/tema8/teste/SimpleDynamicButtonsTests.java:18]: Please add the assertion here. Thanks!

File: src/test/java/madalinapopescu/tests/tema8/teste/RefreshPageTest.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+public class RefreshPageTest extends BaseTest {
+    @Test(groups = {"mobile", "desktop"})
+    public void refreshTest() {
+        driver.get("https://testpages.herokuapp.com/styled/refresh");
+        RefreshPage refreshPage = new RefreshPage(driver);
+        driver.navigate().refresh();
+        Assert.assertEquals(refreshPage.getStatusForTimestamp(), refreshPage.getStatusForTimestamp(), "Timestamp has been changed!");
+    }
+}
+
```
Comment (95teo) [inline src/test/java/madalinapopescu/tests/tema8/teste/RefreshPageTest.java:16]: One blank line is enough. You can remove the extra ones.

File: src/test/java/madalinapopescu/tests/tema8/teste/RedirectPageTest.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    public void clickOnFirstButton() {
+        driver.get("https://testpages.herokuapp.com/styled/javascript-redirect-test.html");
+        WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(5));
+        RedirectPage redirectPage = new RedirectPage(driver);
+        redirectPage.clickOnFirstButton();
+        Assert.assertEquals(driver.getCurrentUrl(),
+                "https://testpages.herokuapp.com/styled/redirected.html");
+        redirectPage.clickOnBackButton();
+    }
+    @Test(groups = {"mobile", "desktop"})
```
Comment (95teo) [inline src/test/java/madalinapopescu/tests/tema8/teste/RedirectPageTest.java:23]: Please leave one blank line between methods. Thanks!

**Review by 95teo — CHANGES_REQUESTED** [review]

File: src/test/java/madalinapopescu/tests/tema8/teste/SimpleDynamicButtonsTests.java (hunk 1 above)
Comment (pmadalina88) [reply src/test/java/madalinapopescu/tests/tema8/teste/SimpleDynamicButtonsTests.java]: L-am adaugat

File: src/test/java/madalinapopescu/tests/tema8/teste/RefreshPageTest.java (hunk 2 above)
Comment (pmadalina88) [reply src/test/java/madalinapopescu/tests/tema8/teste/RefreshPageTest.java:15]: Iar spatii.....

**Review by 95teo — APPROVED** [review]

**Review by ioanaandroflorea — APPROVED** [review]

**Review by curtnazarEliz — APPROVED** [review]

**Pull request merged**